lib/actions/ssh.coffee

```diff
@@ -115,8 +117,8 @@ module.exports =
 					type: 'list'
 					choices: _.map containers, (container) ->
 						return {
-							name: "#{container.name or 'Untitled'} (#{container.id})"
-							value: container.name
+							name: "#{container.Names[0] or 'Untitled'} (#{container.Id})"
```

Comment:
Yup, agree with @lekkas we will only need it when toolbox starts supporting resin.io devices, for now it's just resinOS